src/metax_api/models/common.py

```diff
@@ -151,6 +151,7 @@ def track_fields(self, *fields):
     def _set_removed(self):
         self.removed = True
         self.date_removed = get_tz_aware_now_without_micros()
+        self.date_modified = get_tz_aware_now_without_micros()
```

Comment:
Instead of
``self.date_modified = get_tz_aware_now_without_micros()``
prefer
``self.date_modified = self.date_removed``
to ensure they have the exact same value. Theoreically the former can result in the fields having different values.

(if you want to get fancy, ``self.date_removed = self.date_modified = get_tz...`` may work and be more compact, not sure how it behaves with since those fields are django orm fields)